### PR TITLE
Add kebechet cronjob

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Fixes https://github.com/thoth-station/mi-scheduler/issues/59

## This introduces a breaking change

- [ ] Yes
- [X] No

## Test or Stage Environment

- [ ] Pull Requests added content to STAGE environment
- [ ] Pull Requests to `AICoE/aicoe-cd` has been opened: 

## This Pull Request implements
This PR adds a new cronjob specific for kebechet repositories analysis.

## Description
There should be now two standalone cronjobs, one kebechet and other general GH repo analysis

### CODE REVIEW

File: mi-scheduler/base/cronjob.yaml [hunk 1]
```diff
@@ -2,11 +2,11 @@
 kind: CronJob
 apiVersion: batch/v1beta1
 metadata:
-  name: mi-scheduler
+  name: mi-scheduler-gh
```
Comment (harshad16) [inline mi-scheduler/base/cronjob.yaml:5]: as this is udpate 
the name is to be updated in places in overlays
ex: https://github.com/thoth-station/thoth-application/blob/f095d33200b16b6119889669a77e28381a85d2bc/mi-scheduler/overlays/ocp4-stage/cronjob.yaml#L5

Comment (xtuchyna) [reply mi-scheduler/base/cronjob.yaml:5]: done in all overlays

**Review by harshad16 — APPROVED** [review]
/approve

this can go in.

**Pull request merged**